configure.ac

```diff
@@ -224,12 +224,48 @@ then
 fi
 AC_SUBST([ENABLED_ASM])
 
+ENABLED_FIPS="no"
+# wolfEngine Options
+# This requires a FIPS source code bundle.
```

Comment:
I think this comment is incorrect. You can build wolfSSL for use with wolfEngine from a non-FIPS bundle.

Reply:
Good call. Removed.